packages/hint-ie-flexbox-compat/src/hint.ts

```diff
@@ -0,0 +1,84 @@
+import { Category } from '@hint/utils-types';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
+import { HintMetadata } from 'hint/dist/src/lib/types';
+import { IHint } from 'hint/dist/src/lib/types';
+import { getFullCSSCodeSnippet, getCSSLocationFromNode } from '@hint/utils-css';
+import { StyleEvents, StyleParse } from '@hint/parser-css';
+import { Severity } from '@hint/utils-types';
+
+import { getMessage } from './i18n.import';
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+export default class IEFlexboxCompatHint implements IHint {
+    public static readonly meta: HintMetadata = {
```

Comment:
Meta needs to be in its own file (`meta.ts`) and imported. (e.g. https://github.com/webhintio/hint/blob/main/packages/hint-manifest-is-valid/src/hint.ts).
We use the meta file to generate the documentation in the website.

Reply:
Thanks! Done.